spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/utils/SparkUtils.scala

```diff
@@ -89,6 +89,26 @@ object SparkUtils extends Logging {
     catalog
   }
 
+  /**
+   * Get the catalog table of current active SparkSession if needed.
+   */
+  def getCatalogTableIfNeeded(tableDefinition: CatalogTable): CatalogTable = {
+    // Let's assume that input catalog table does not have full information
+    // if both owner and last access time are default values (empty string and -1)
+    if (tableDefinition.owner == "" && tableDefinition.lastAccessTime == -1) {
```

Comment:
Do we need this? I guess `try .. catch` blindly might be enough.

Reply:
yup. addressed.